.github/workflows/snap.yml

```diff
@@ -0,0 +1,23 @@
+name: Pebble snap
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Pebble repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build Pebble Snap
+        id: build-pebble-snap
+        uses: snapcore/action-build@v1
+
+      - name: Attach pebble snap to GH workflow execution
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ steps.build-pebble-snap.outputs.snap }}
+          path: ${{ steps.build-pebble-snap.outputs.snap }}
```

Comment:
Will we be adding `snapcore/action-publish` to publish it to Charmhub too?

Reply:
To Charmhub or the snap store? The latter will be done from the store itself, and promotions will be done here (coming in a future PR)

Reply:
Got it. Yeah, sorry, I meant the Snap store. I live in the charm world. :-)